sonar-project.properties

```diff
@@ -39,11 +39,14 @@ sonar.swift.simulator=platform=iOS Simulator,name=iPhone 6,OS=9.2
 #sonar.swift.workspace=MyWrkSpc.xcworkspace
 # Specify your appname.
 # This will be something like "myApp"
-# Use when basename is different from targeted scheme. 
+# Use when basename is different from targeted scheme.
 # Or when slather fails with 'No product binary found'
 # sonar.swift.appName=myApp
 # Scheme to build your application
 sonar.swift.appScheme=MyScheme
+# Specify a list of framework names.
+# Use if you want to measure code coverage with slather on them.
+# sonar.coverage.otherBinaryNames=myFramework,myOtherFramework
```

Comment:
Why not simply using `sonar.swift.binaryNames`?
Using this name we clearly understandard what content it should carry.
Should be located in the `Optional configuration` section, since it is optional.